### ISSUE TITLE
HTML 5 Panel ?

### ISSUE THREAD

**Issue by SheepDomination** [seblavoie/adobe-illustrator-layer-renamer#3]
Will this ever be a HTML 5 panel for Illustrator ?

**Comment by seblavoie** [comment]
I don't think it is a necessity for my personal needs and I don't have much energy I want to invest in rebuilding this script with new technologies, but I am surely open if someone wants to take it to another level. Feel free to submit a pull request.
